specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2025-03-01/costmanagement.exports.json

```diff
@@ -0,0 +1,968 @@
+          "type": "string",
+          "format": "date-time"
+        }
+      },
+      "required": [
+        "from",
+        "to"
+      ]
+    },
+    "ExportExecutionListResult": {
+      "description": "Result of listing the run history of an export.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "A list of export runs.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ExportRun"
+          }
+        }
+      }
+    },
+    "ExportRun": {
+      "description": "An export run.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "common-types.json#/definitions/CostManagementProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ExportRunProperties",
+          "title": "Export run properties"
+        }
+      }
+    },
+    "ExportRunProperties": {
+      "description": "The properties of the export run.",
+      "type": "object",
+      "properties": {
+        "executionType": {
+          "description": "The type of the export run.",
+          "type": "string",
+          "enum": [
+            "OnDemand",
+            "Scheduled"
+          ],
+          "x-ms-enum": {
+            "name": "ExecutionType",
+            "modelAsString": true
+          }
+        },
+        "status": {
+          "description": "The last known status of the export run.",
+          "type": "string",
+          "enum": [
+            "Queued",
+            "InProgress",
+            "Completed",
+            "Failed",
+            "Timeout",
+            "NewDataNotAvailable",
+            "DataNotAvailable"
+          ],
+          "x-ms-enum": {
+            "name": "ExecutionStatus",
+            "modelAsString": true
+          }
+        },
+        "submittedBy": {
+          "description": "The identifier for the entity that triggered the export. For on-demand runs it is the user email. For scheduled runs it is 'System'.",
+          "type": "string"
+        },
+        "submittedTime": {
+          "description": "The time when export was queued to be run.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "processingStartTime": {
+          "description": "The time when export was picked up to be run.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "processingEndTime": {
+          "description": "The time when the export run finished.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "startDate": {
+          "description": "The start datetime for the export.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "endDate": {
+          "description": "The end datetime for the export.",
+          "type": "string",
+          "format": "date-time"
+        },
```

Comment:
will this also capture time in the value ? 
If so, can these be startTimeInUTC and endTimeInUTC ? 

Please mention the timezone in description.